source/class/qx/ui/form/AbstractSelectBox.js

```diff
@@ -293,7 +305,7 @@ qx.Class.define("qx.ui.form.AbstractSelectBox",
       }
 
       // hide the list always on escape
-      else if (!listPopup.isHidden() && identifier == "Escape")
+      else if (!listPopup.isHidden() && (identifier == "Escape" || identifier == "Tab"))
```

Comment:
This is a change of general behavior, not just ARIA attribute-related. I wonder if this is appropriate???

Reply:
This is removable. I added it at my first tries because i made the list and its elements focusable too. In the current version this is not needed anymore, but i kept it because i liked it and saw it in another framework too.
Closing SelectMenus with Tab is not a W3C requirement.